virt/virt-2-4-release-notes.adoc

```diff
@@ -102,6 +102,8 @@ from being inadvertently started before an upload has completed.
 == Known issues
 //https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
 //https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
+* If you upgrade to {VirtProductName}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
```

Comment:
One more thing -- "If you upgrade to {VirtProductName}" should probably either state the version ("If you upgrade to {VirtProductName} {VirtVersion}") or just say "If you upgrade {VirtProductName}"

And I think there should be a space between "...to avoid issues." and the BZ link.

Reply:
Content updated.